tests/core/test_expectation_configuration.py

```diff
@@ -88,8 +88,8 @@ def test_expectation_configuration_equality(config1, config2, config3, config4):
     assert config1 is config1  # no difference  # noqa: PLR0124
     assert config1 is not config2  # different instances, but same content
     assert config1 == config2  # different instances, but same content
-    assert not (config1 != config2)  # ne works properly
-    assert not (config1 == config3)  # different meta
+    assert config1 == config2  # ne works properly
+    assert config1 != config3  # different meta
```

Comment:
Need to revert this one, but I don't think this means we need to disable the rule.